docs/integrations/integration-catalog.md

```diff
@@ -1,8 +1,12 @@
 ---
-title: 'Add to the addon catalog'
+title: 'Add to the Integration catalog'
```

Comment:
We typically use "Sentence case" for our headings. Though I can see a case for capitalizing key terms like this. Can you explain your reasoning here?